input/lightcurves/DESYLCArchiveToECSV.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+from astropy.table import Table, Column
+from astropy.units import cds
+import string
+
+def main(a, b):
+	## open file
```

Comment:
Python has a style guide (https://www.python.org/dev/peps/pep-0008/).
One of the things it says is that Python code should be indented with 4 spaces.

There's many tools that auto-format Python code.
I use https://www.jetbrains.com/pycharm/ to write the code and just hit the keyboard shortcut for  "Code -> Reformat code" before I save, that auto-formats it.
If you do Python coding, I highly recommend you get a good editor or IDE that does this for you.
The free PyCharm version is IMO the best choice.